app/gateway/2.7.x/install-and-run/upgrade-enterprise.md

```diff
@@ -20,68 +20,60 @@ distinction between major, minor, and patch versions. The upgrade
-* If running a version of {{site.ee_product_name}} earlier than 2.3,
-  [migrate to 2.3](/enterprise/2.3.x/deployment/upgrades/migrations/) first.
-* If running a version of {{ site.ee_product_name }} earlier than 2.4,
-  [migrate to 2.4](/enterprise/2.4.x/deployment/upgrades/migrations/) first.
-* If running a version of {{ site.ee_product_name }} earlier than 2.5,
-  [migrate to 2.5](/enterprise/2.5.x/deployment/upgrades/migrations/) first.
+## Upgrade considerations
 
-### Kong Manager upgrade to 2.7.x
-
-Version 2.7.x introduced a new way to configure the OIDC plugin to map IdP roles to Kong Manager admin accounts. 
-You must now specify the `admin_claim` instead of the `consumer_claim` in your OIDC config file. For more information, 
-see [OIDC Authenticated Group Mapping](/gateway/2.7.x/configure/auth/kong-manager/oidc-mapping/).
+Before upgrading, review this list for any configuration or breaking changes that
+affect your current installation.
 
-### Dev Portal migrations
+If you are adding a new plugin to your installation, you need to run
+`kong migrations up` with the plugin name specified. For example,
+`KONG_PLUGINS=oauth2`.
 
-There are no migrations necessary for the Dev Portal when upgrading from 2.5.x to
-2.6.x.
+### Kong Manager breaking changes
 
-If you are currently using the Developer Portal in 1.5.x, it will no longer work without
-[manually migrating files](/enterprise/2.1.x/developer-portal/latest-migrations) to version 2.1.x.
+Version 2.7.x introduced a new way to configure the OIDC plugin to map IdP roles to Kong Manager admin accounts.
+You must now specify the `admin_claim` instead of the `consumer_claim` in your OIDC config file. For more information,
+see [OIDC Authenticated Group Mapping](/gateway/{{page.kong_version}}/configure/auth/kong-manager/oidc-mapping/).
 
-### Upgrade considerations
+### Dev Portal migrations
 
-Before upgrading, review this list for any configuration or breaking changes that
-affect your current installation.
+There are no migrations necessary for the Dev Portal when upgrading from 2.6.x to
+2.7.x.
 
-* If you are adding a new plugin to your installation, you need to run
-  `kong migrations up` with the plugin name specified. For example,
-  `KONG_PLUGINS=oauth2`.
+If you are currently using the Dev Portal in 1.5.x or earlier,
+[manually migrate the files](/enterprise/2.1.x/developer-portal/latest-migrations)
+to version 2.1.x before continuing.
 
 ### Hybrid mode considerations
 
 {:.important}
-> **Important:** If you are currently running in [hybrid mode](/enterprise/{{page.kong_version}}/deployment/hybrid-mode/),
-upgrade the Control Plane first, and then the Data Planes.
+> **Important:** If you are currently running in [hybrid mode](/gateway/{{page.kong_version}}/plan-and-deploy/hybrid-mode/),
+upgrade the control plane first, and then the data planes.
 
-* If you are currently running 2.6.x in classic (traditional)
+* If you are currently running 2.7.x in classic (traditional)
```

Comment:
It seems to me that "2.6" was right in this context. This phrase was meant for users who are using traditional mode, want to upgrade and start using hybrid mode at the same time, right?

Reply:
Nice catch, you're right. I'll change this one back.